### PR TITLE
feat: bzz.link support

### PR DESCRIPTION
**Pull request by AuHau**
Adds Bzz.link support on two places:

 - when you upload a website there is "Bzz link tab" instead of "Web link" with a button to open the bzz.link
 - when you access a website reference then the "Open website" now is "Open bzz link" with the bzz.link

### CODE REVIEW

**Comment by bee-worker** [comment]
## 🐝 PR preview in Swarm

**Preview URL:** https://bafybwidlttqmpgo2qjsjtlxhebcln6yql3t6ntuqgu22clpjfsjaabudcy.bzz.link
**Swarm Hash:** `6b9ce0c799da826499aee72044b6fb105ee7e6ce903535a12de92c9200068316`
**Commit Hash:** 9e6e9ff2e1c8e41fcc922a560914888b671d6d08
**Commit Message:** chore: wording

File: src/constants.ts [hunk 1]
```diff
@@ -12,3 +12,4 @@ const url = window.location.origin
 
 export const GATEWAY_URL = process.env.REACT_APP_GATEWAY_URL || url
 export const DIRECT_DOWNLOAD_URL = process.env.DIRECT_DOWNLOAD_URL || 'https://download.gateway.ethswarm.org/bzz/'
+export const BZZ_LINK_DOMAIN = process.env.BZZ_LINK_DOMAIN || 'bzz.link'
```
Comment (vojtechsimetka) [inline src/constants.ts:15]: can you please add this to the README?

**Review by vojtechsimetka — APPROVED** [review]

**Comment by agazso** [comment]
> * when you upload a website there is "Bzz link tab" instead of "Web link" with a button to open the bzz.link
> * when you access a website reference then the "Open website" now is "Open bzz link" with the bzz.link

I suggest to not change the text at least not on the upload page, because it causes inconsistency with the rest of the page, also at this point it may be not very clear what is a bzz.link.

For example here the title says `Share your file with a simple web link, or a Swarm hash.` but below there is no web link to choose from, only bzz.link instead.
Screenshot 2021-12-20 at 16 37 42

**Comment by AuHau** [comment]
@agazso but Bzz link is a web link so I don't think that is a problem... This comes from a design that David made...

**Comment by agazso** [comment]
> @agazso but Bzz link is a web link so I don't think that is a problem... This comes from a design that David made...

The design is a bit different because it already uses the page for accessing a page as for uploading, therefore the text is a bit different. I don't think this is a huge issue, we may deal with it later.

**Pull request merged**